### PR TITLE
N lopt

### PR DESCRIPTION
**Pull request by nikishe**
NLOPT LIB doc stub

### CODE REVIEW

File: FAQs.rst
Comment (jkwmoore) [inline FAQs.rst:1]: This looks like an accidental inclusion?

File: stanage/software/libs/nlopt
Comment (jkwmoore) [inline stanage/software/libs/nlopt:1]: Can we just add a section for the install notes (as per other pages)  indicating the installation of this library is generated by Easybuild as a dependency of R?

Other than this, LGTM.

File: stanage/software/libs/nlopt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+   module load NLopt/2.6.1-GCCcore-9.3.0
+   module load NLopt/2.6.2-GCCcore-10.2.0
+   module load NLopt/2.7.0-GCCcore-11.2.0
+   module load NLopt/2.7.1-GCCcore-11.3.0
+   module load NLopt/2.7.1-GCCcore-12.2.0
+
+Installation method
+^^^^^^^^^^^^^^^^^^^
+
+This was installed as an Easybuild dependancy for `R <https://docs.hpc.shef.ac.uk/en/latest/stanage/software/apps/R.html/>`_
```
Comment (Carldkennedy) [inline stanage/software/libs/nlopt:27]: dependency

**Review by Carldkennedy — COMMENTED** [review]
Small correction. Page is not rendering on the website

**Comment by jkwmoore** [comment]
stanage/software/libs/nlopt =/= stanage/software/libs/nlopt.rst - please fix.

**Pull request merged**